src/Epic/index.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactElement } from 'react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { Epic } from './index';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import type { EpicProps } from './index';
+import { knobsData } from '../utils/knobsData';
+
+export default {
+    component: 'Epic',
+    title: 'Components/Epic',
+    decorators: [withKnobs],
+    parameters: {
+        knobs: {
+            escapeHTML: false, // Block HTML escaping, preventing double-escaping of imgUrl special characters in Storybook
```

Comment:
I wonder if this is needed here, since it relates to the `imgUrl` which isn't a prop of the epic.

Reply:
Good point, will remove it.